docs/json_schemas/grafana_datasource_exchange/v0/requirer.json

```diff
@@ -0,0 +1,77 @@
+{
+  "$defs": {
+    "BaseModel": {
+      "properties": {},
+      "title": "BaseModel",
+      "type": "object"
+    },
+    "GrafanaDatasource": {
+      "properties": {
+        "type": {
+          "description": "Type of the datasource.",
+          "examples": [
+            "tempo",
```

Comment:
Same as above: I wonder if these shouldn't be `traces`, `logs`, `metrics` not to focus on a specific tool we use. However this won't change the type itself.